content/_guides/creatingchannels.md

```diff
@@ -1,52 +1,196 @@
 ---
 title: Creating Channels
 category: chanop
-credits: web7
 ---
 
+A person who joins an _empty_ and unregistered channel is granted
+**temporary** Operator status. This does not guarantee their ownership of the
+channel. They need to `REGISTER` the channel with `ChanServ` to gain ownership
+of it.
+
+This guide will help you prepare for and go through that process.
+
 ## Choosing the correct namespace
 
 Before creating a new channel on Libera.Chat, please take a moment to read up
 on [our namespaces policy](/chanreg) and ensure that your channel is named
 appropriately from the beginning. This will avoid disruptions in the future.
 
-## How to create a new channel
+## Be identified
 
-### Be identified
+To register and manage channels, you will need to be [identified to](registration#logging-in)
+`NickServ`.
 
-You must be registered and logged in to services to make channels. You can log
-in by doing `/msg NickServ IDENTIFY YourNickname YourPassword`
+## Pick a name
 
-### Joining the channel
+Before doing that, you'll obviously need to pick a channel name! Think of one
+that is descriptive of the channel's purpose, and has no spaces or commas.
+The name you choose will become what's called your namespace. You can use
+some punctuation such as `.`, `_` or `+` instead of spaces, but it is better
+to just drop the space. It is typical to reserve `-` as a delimiter for
```

Comment:
Do you think it's worth saying something stronger here. If someone's making a project channel that `-` will be a namespace delimiter. It's good to avoid them for the root channel name in these cases, and having one can slow down or fail a namespace assignment.

Reply:
Yeah i've added a bit about namespace conflicts